src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/ApprovalController.java

```diff
@@ -44,6 +44,7 @@
     private static final String IS_PAYABLE_TRANSACTION = "isPayableTransaction";
 
     private static final UriTemplate SUBMITTED_ACCOUNTS_PATH = new UriTemplate("/company/{companyNumber}/transaction/{transactionId}/company-accounts/{companyAccountsId}/small-full/approved-accounts");
+    private static final String SUMMARY_SCREEN_FALSE = "?summary=false";
```

Comment:
keeping it consistent with other apps please could you update
SUMMARY_FALSE_PARAMETER

Reply:
Changed variable name for consistency